src/cache/cache_las.c

```diff
@@ -557,6 +557,7 @@ __las_squash_modifies(WT_SESSION_IMPL *session, WT_CURSOR *cursor, WT_UPDATE **u
     size_t allocated_bytes;
     u_int i;
 
+    WT_CLEAR(las_value);
```

Comment:
Yes, I agree with it, Earlier the initialization was close to the usage place, but it leads to clang scan-build warnings of uninitialized variable usage. The `las_value` is freed in the `err` path whenever there is any error occurs. The uninitialized warning is coming with an assumption of an error occurring before the variable is initialized. Because of that reason, I moved the initialization to a place before any error occurs.